docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -58,6 +59,17 @@ the implementation without compat concerns in future releases.
   - Add new test code following [conventions](project-guidelines.md#code-file-naming-conventions) for new files to that are specific to the new target framework.
   - To run just the new test configuration run `dotnet msbuild <Library>.csproj /t:RebuildAndTest /p:TargetGroup=<TargetGroup>`
 
+## Documentation
+
+New public APIs must be documented with triple slash comments on top of them. Visual Studio automatically generates the structure for you when you type `\\\`.
```

Comment:
I thought `///` is what Visual Studio recognized for automatically generating the structure.

Reply:
```suggestion
New public APIs must be documented with triple-slash comments on top of them. Visual Studio automatically generates the structure for you when you type `///`.
```